package.json

```diff
@@ -47,8 +44,14 @@
     "webmaker-app-icons": "git://github.com/k88hudson/ionicons#teach",
     "webpack": "^1.6.0"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "mocha-phantomjs": "^3.5.3",
+    "phantomjs": "^1.9.16",
+    "simplecrawler": "^0.5.2",
+    "supertest": "^1.0.1"
+  },
   "scripts": {
+    "postinstall": "gulp postinstall",
```

Comment:
I had to add this post-install to generate the webpack bundle (only when `NODE_ENV=production` though, otherwise it does nothing) because it takes so long to generate the bundle that I couldn't just do it on server startup--when I tried, Heroku raised a [Boot timeout error](https://devcenter.heroku.com/articles/error-codes#r10-boot-timeout) and stopped the app.

By generating the static files in a `postinstall` step, it actually becomes part of the [slug](https://devcenter.heroku.com/articles/slug-compiler) that Heroku uses to launch an instance of the server--so it's actually a good thing, because it means spinning up new servers takes way less time, thus allowing us to scale faster.